resources/pvc.py

```diff
@@ -0,0 +1,67 @@
+"""
+General PVC object
+"""
+import tempfile
+import logging
+
+from resources.ocs import OCS
+
+log = logging.getLogger(__name__)
+
+
+class PVC(OCS):
+    """
+    A basic PersistentVolumeClaim kind resource
+    """
+
+    def __init__(self, **kwargs):
+        """
+        Initializer function
+        kwargs:
+            See parent class for kwargs information
+        """
+        super(PVC, self).__init__(**kwargs)
+        self.temp_yaml = tempfile.NamedTemporaryFile(
+            mode='w+', prefix='PVC_', delete=False
```

Comment:
Not sure why this is necessary here. In OCS() it does the same thing:
```python
        self.temp_yaml = tempfile.NamedTemporaryFile(
            mode='w+', prefix=self._kind, delete=False
        )
```

Reply:
True true